segmentation/include/pcl/segmentation/impl/unary_classifier.hpp

```diff
@@ -309,7 +307,8 @@ pcl::UnaryClassifier<PointT>::queryFeatureDistances (std::vector<pcl::PointCloud
   {
     // Query point  
     flann::Matrix<float> p = flann::Matrix<float>(new float[n_col], 1, n_col);
-    memcpy (&p.ptr ()[0], (*query_features)[i].histogram, p.cols * p.rows * sizeof (float));
+    std::copy_n(&(query_features->at(i).histogram[0]), p.cols * p.rows,
```

Comment:
Is `at` needed?

Reply:
Probably not (although there was a `memcpy` before); removed